### ISSUE TITLE
Add focus state for Back-to-Top link. 

### ISSUE THREAD

**Issue by richardgaunt** [civictheme/uikit#406]
### User story

As a developer
I want to be able to add focus state to the back to top button
So that override styles

### Description

UIKit already had a focus state, but it was set to transparent as per designs - the design does not have a focus state.

We need to add a focus state to uikit so it can be overridden

### Acceptance criteria

Given I am a developer
When I override back to top component styles
Then I expect to be able to change a variable to change the focus state

**Comment by sonamchaturvedi28** [comment]
Test Env: DEV
Test Status: PASS
Test Result:
- Focus state is added to the back to top button
<img width="636" alt="Screenshot 2024-10-26 at 12 10 59 AM" src="https://github.com/user-attachments/assets/0733f184-ded7-4849-9b7b-4768d9bf2b0f">

<img width="909" alt="Screenshot 2024-10-26 at 12 23 25 AM" src="https://github.com/user-attachments/assets/c767141f-5083-4d41-9e56-871f71d2ed05">
